lib/css_parser/rule_set.rb

```diff
@@ -109,10 +125,8 @@ def each_selector(options = {}) # :yields: selector, declarations, specificity
 
     # Iterate through declarations.
     def each_declaration # :yields: property, value, is_important
-      decs = @declarations.sort { |a, b| a[1][:order].nil? || b[1][:order].nil? ? 0 : a[1][:order] <=> b[1][:order] }
-      decs.each do |property, data|
-        value = data[:value]
-        yield property.downcase.strip, value.strip, data[:is_important]
+      @declarations.sort_by { |_name, value| value.order }.each do |property_name, property|
```

Comment:
calling the same thing different names is confusing
```suggestion
      @declarations.sort_by { |_, value| value.order }.each do |property_name, value|
```